### PR TITLE
Add 07/2020 release announcement

### PR DESCRIPTION
**Pull request by aahlenst**

### CODE REVIEW

File: content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+You can then choose between these auto-upgrading casks and install them with `brew cask install <name>`: 
+
+* adoptopenjdk
+* adoptopenjdk-jre
+* adoptopenjdk-openj9
+* adoptopenjdk-openj9-jre
+
+### Eclipse OpenJ9 0.21.0
```
Comment (gdams) [inline content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md:60]: probably no need for this (the release notes above summarises the changes

File: content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+* adoptopenjdk
+* adoptopenjdk-jre
+* adoptopenjdk-openj9
+* adoptopenjdk-openj9-jre
+
+## A Note on Binaries for 32-bit ARM
+
+Because we're short on machines with 32-bit ARM CPUs and those we have are also incredibly slow, our testing capacity is minimal. As a result, we haven't caught a bug in the last release that caused certain SSL connections to fail. This has been fixed in the meantime. We are already looking into procuring additional capacity that should make it possible to re-enable our entire test suite and deliver binaries that match our quality criteria.
+
+Another longstanding problem is that AdoptOpenJDK 8 on 32-bit ARM only supports interpreted mode (Zero VM). We would very much like to fix that. Hence, we're looking for volunteers who help us build AdoptOpenJDK 8 with a proper client VM that is already available on AdoptOpenJDK 11 and higher.
```
Comment (gdams) [inline content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md:64]: @sxa is this correct? I thought that we were building with a JIT on arm32 now

Comment (aahlenst) [reply content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md:64]:
```
jdk8u262-b09/bin/java -version
openjdk version "1.8.0_262"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_262-202007131722-b09)
OpenJDK Zero VM (AdoptOpenJDK)(build 25.262-b09, interpreted mode)
```

Comment (aahlenst) [reply content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md:64]: It's the last nightly build, however.

Comment (gdams) [reply content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md:64]: yes you're right - sorry we reverted (https://github.com/AdoptOpenJDK/openjdk-build/commit/4937941d0d2285a191d35824b43a982d76e08a29)

I'll kick off a zero arm32 build now then

File: content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md (hunk 1 above)
Comment (aahlenst) [reply content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md]: Yeah, I dropped it while you were reviewing. There's nothing very interesting in the release notes.

File: content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+title: AdoptOpenJDK 8u262, 11.0.8, and 14.0.8 Available
+date: "2020-07-16T12:00:00+00:00"
+author: andreasahlenstorf
+description: AdoptOpenJDK is happy to announce the immediate availability of AdoptOpenJDK 8u262, 11.0.8, and 14.0.2, both with OpenJDK HotSpot and Eclipse OpenJ9. As always, all binaries are thoroughly tested and available free of charge without usage restrictions on a wide range of platforms.
```
Comment (karianna) [inline content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md:5]: `AdoptOpenJDK is happy to announce the immediate availability of AdoptOpenJDK 8u262, 11.0.8, and 14.0.2, both with OpenJDK HotSpot and Eclipse OpenJ9. As always, all binaries are thoroughly tested and available free of charge without usage restrictions on a wide range of platforms.`

-->

`AdoptOpenJDK is happy to announce the immediate availability of AdoptOpenJDK 8u262, 11.0.8, and 14.0.2.  Binaries are immediately available for OpenJDK and Eclipse Op[enJ9 (with OpenJDK class libraries). As always, all binaries are thoroughly tested and available free of charge without usage restrictions on a wide range of platforms.`

File: content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+tags:
+  - announcement
+---
+
+AdoptOpenJDK is happy to announce the immediate availability of AdoptOpenJDK 8u262, 11.0.8, and 14.0.2, both with [OpenJDK HotSpot](https://openjdk.java.net/groups/hotspot/) and [Eclipse OpenJ9](https://www.eclipse.org/openj9/). As always, all binaries are thoroughly tested (except 32-bit ARM, see below) and available free of charge without usage restrictions on a wide range of platforms.
```
Comment (karianna) [inline content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md:10]: Same suggested change as above

**Comment by aahlenst** [comment]
@karianna @gdams Please have another look. I've changed the intros.

**Pull request merged**